src/server/server.js

```diff
@@ -218,13 +219,35 @@ SimulationServer.prototype._streamAppHostHtml = function (request, response) {
                 return '<script src="' + scriptSource + '"></script>';
             }).join('');
 
-            // Note we replace "default-src 'self'" with "default-src 'self' ws:" (in Content Security Policy) so that
-            // websocket connections are allowed (this relies on a custom version of send that supports a 'transform' option).
+            // Note we add "connect-src 'self' ws:" and "img-src 'blob:' (in Content Security Policy) so that
+            // websocket connections are allowed and the camera plugin works (this relies on a custom version
```

Comment:
My understanding is that the camera plugin works by providing `blob:` URLs, and that's the reason for adding it to the CSP.